gp-nested-forms/gpnf-sync-parent-child-entry-payment-status.php

```diff
@@ -92,6 +92,9 @@ function gpnf_has_product_field( $form ) {
 
 if ( ! function_exists( 'gpnf_get_parent_entry_and_sync_child_entries_payment_details' ) ) {
 	function gpnf_get_parent_entry_and_sync_child_entries_payment_details( $entry_id ) {
+		if ( ! is_scalar( $entry_id ) ) {
```

Comment:
@saifsultanc What's the scenario where this not be scalar?

Reply:
@spivurno It trigerred via the hook `gform_after_update_entry` call [here](https://github.com/gravitywiz/snippet-library/blob/master/gp-nested-forms/gpnf-sync-parent-child-entry-payment-status.php#L25)

Here is the debug snapshot:
<img width="1719" alt="Screenshot 2024-02-23 at 10 40 16 PM" src="https://github.com/gravitywiz/snippet-library/assets/26293394/d9666f8a-f6f2-45a5-a7b4-29c5c568d720">

Look at the data content on the `entry_id`

Reply:
@saifsultanc Ah, I see. Thoughts: https://www.loom.com/share/5bc1c38e8b4544498b57b676535d52cd